app/views/search/_collection_details.html.erb

```diff
@@ -94,16 +94,25 @@
               <!-- /ko -->
             </dd>
           </dl>
+          <!-- end of second row -->
+
+          <ul>
+            <!-- ko foreach: $data.highlighted_urls -->
+            <li><a data-bind="attr: {href: $data.url}, text: $data.name" target="_blank"></a></li>
+            <!-- /ko -->
+            <!-- ko if: $data.related_urls.length > 0 -->
```

Comment:
If I do this here then there will be an empty `<li>` element when there are no related urls. I'm also just not sure why, the file uses `<!-- ko if .... -->` all throughout